### DISCUSSION TITLE
Is Aurora Serverless only available for Prisma2 from within the same VPC?

### DISCUSSION THREAD

**Discussion by kennandavison** [prisma/prisma#2209] [Q&A] [answered]
I've read:
https://hasura.io/blog/exploring-hasura-as-a-data-access-layer-in-serverless-architecture/
https://v1.prisma.io/forum/t/cant-connect-to-serverless-aurora-with-prisma-cloud/5532/5
https://github.com/prisma/prisma/issues/885
https://github.com/prisma/prisma/issues/276
https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/deployment#serverless-faas
https://github.com/prisma/prisma/issues/1297

As of today, do you still need Prisma2 to be in the same VPC as Aurora Serverless to work (excluding public IP)? 

Any word on Prisma2 Aurora Serverless Data API support?

**Comment by ryands17** [comment, marked as answer]
Hey @kennandavison 
Currently is would only be available within the same VPC. To use it locally, you would have to create a VPN for the timebeing.

We are looking into support for Serverless Data API but there's no concrete timeline for that.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
